templates/post.php

```diff
@@ -1,19 +1,16 @@
 <?php
-$wwwlib = sprintf(
-    "//%s/%sresources/", 
-    htmlentities($_SERVER['SERVER_NAME']), 
-    htmlentities($this->data['baseurlpath'])
-);
+use SimpleSAML\Utils\HTTP;
 ?>
+
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
         "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
 <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
 <head>
     <meta http-equiv="content-type" content="text/html; charset=utf-8" />
     <title>POST data</title>
-    <script type="text/javascript" src="<?php echo $wwwlib;?>post.js"></script>
+    <script type="text/javascript" src="<?php echo HTTP::getBaseURL();?>/resources/post.js"></script>
```

Comment:
This is wrong. According to `HTTP::getBaseURL()` documentation:

```php
    /**
     * Retrieve the base URL of the SimpleSAMLphp installation. The URL will always end with a '/'. For example:
     *      https://idp.example.org/simplesaml/
```

This means the URL generated here would be "whatever//resources/post.js".

Additionally, there's no need to use the functions from `SimpleSAML\Utils\HTTP` directly, as the base URL is already available in the data passed to the template. Instead of this, the code should read:

```php
    <script type="text/javascript" src="/<?php echo $this->data['baseurlpath']; ?>resources/post.js"></script>
```